etc/profile-m-z/quodlibet.profile

```diff
@@ -0,0 +1,51 @@
+# Firejail profile for quodlibet
+# Description: Music player and music library manager
+# This file is overwritten after every install/update
+# Persistent local customizations
+include quodlibet.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${MUSIC}
+noblacklist ${HOME}/.cache/quodlibet
+noblacklist ${HOME}/.config/quodlibet
+noblacklist ${HOME}/.quodlibet
+
+whitelist ${MUSIC}
+whitelist ${HOME}/.cache/quodlibet
+whitelist ${HOME}/.config/quodlibet
+whitelist ${HOME}/.quodlibet
```

Comment:
- Add `mkdir`/`mkfile`s
- Add `include whitelist-common.inc`
- Move the whitelist down (between `include disable-*.inc` and `apparmor`)
- Can we add `whitelist-runuser-common.inc`, `whitelist-var-common.inc`, `whitelist-usr-share-common.inc`?